### PR TITLE
Competition page

### PR DESCRIPTION
**Pull request by MPDR200011**
# Description

Page for the programming competition, just a change in the order of elements. Was going to create this PR only when we had the new data, but since other pages are being added without content I decided to create the PR for the page now and add the new content later.

# Tasks

- [x] Change the layout
- [x] Fix spacings
- [ ] Add new regulation text
- [ ] Add new prize information
- [ ] Add new hackerank link

### CODE REVIEW

**Review by SocksDevil — APPROVED** [review]
Lgtm but there is many commits here that have no value whatsoever. Might we squash all of these commits on merge?

**Review by SocksDevil — CHANGES_REQUESTED** [review]
In mobile this happens: 
image

Shouldn't all of the prizes to show in the yellow section?

**Comment by MPDR200011** [comment]
> Shouldn't all of the prizes to show in the yellow section?

In that case I copied the behavior of what happened in last year's version of the page.

image

I can change it if required.

**Comment by MPDR200011** [comment]
> Lgtm but there is many commits here that have no value whatsoever. Might we squash all of these commits on merge?

Yes, we can do this.

File: src/pages/competicao-programacao.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ import PageBanner from "../components/utils/page_banner"
 const CompetitionPage = () => (
   <Layout>
```
Comment (pedromsfernandes) [inline src/pages/competicao-programacao.js]: Should be a dark footer

**Review by pedromsfernandes — CHANGES_REQUESTED** [review]

**Review by SocksDevil — DISMISSED** [review]
:ship:

**Review by pedromsfernandes — DISMISSED** [review]

**Comment by SantosAntero** [comment]
Everything looks fine, but I would like to point out that the "Participar" button seems really small compared with the bar and the mockups, I don't know if this is something that I didn't notice before or if some change affected this. I understand this can be considered a bit of scope creep 👀 but since everything else is fine I don't think it would be that bad if we fixed that in this PR.

**Comment by MPDR200011** [comment]
Ok, I updated the participate button style to be bigger. I also added the text that I completely forgot to add.

**Review by SocksDevil — DISMISSED** [review]
Thanks @SantosAntero ! It looks fine to me now, is it for you?

**Comment by MPDR200011** [comment]
I went ahead and also added a breakpoint to the button as it was becoming deformatted on smaller screens.

**Review by pedromsfernandes — APPROVED** [review]

**Review by SantosAntero — APPROVED** [review]
Looks good!

**Pull request merged**